zhaquirks/tuya/ts130f.py

```diff
@@ -167,6 +167,65 @@ class TuyaZemismartTS130F(CustomDevice):
     }
 
 
+class TuyaOxtTS130F(CustomDevice):
```

Comment:
All the Tuya quirks naming is a nightmare...
I would suggest name this as `TuyaTS130FTOGP` (or whatever without the vendor reference).